### ISSUE TITLE
How to determine results of owned probe in output

### ISSUE THREAD

**Issue by Ernst2020** [jsdelivr/globalping-cli#128]
Hi,

If I do a measurement, how do I determine the results of my probe in the output? 

For instance if I do a ping measurement towards a certain IP using all 106 probes in The Netherlands,  how do I compare the results of my probe to the other 105?

Currently my probe is the only one in the city Globalping assigned it so I can filter on city, but what if another probe is assigned to that city?

In the list of all active probes there is a "tags" field in the json output. How can I add a tag? The tags don't seem to be in the output of the CLI but is it possible to use tags for this purpose perhaps?

Regards,

Ernst

**Comment by jimaek** [comment]
If I understand correctly you would like your own probes to be somehow marked in the results of CLI?
At the moment you would have to manually identify it using the metadata like ASN, but it sounds like an interesting idea.

We will soon launch our user dashboard, you could then adopt your probes by proving ownership. Then if you authenticate the CLI with your API key the API would know which probes are yours and we could visually mark them in the CLI, as well as add tags in the raw JSON responses.

Would that be helpful?

**Comment by Ernst2020** [comment]
Yes. Any functionality that would add a way to identify my probe in output of the CLI would help to compare results.

**Comment by MartinKolarik** [comment]
It's an interesting idea, but the probe "ownership" is not public information at the moment. If we were to add this automatically for all measurements, I wonder if it wouldn't raise privacy concerns for some people. It could perhaps be a user setting?

**Comment by jimaek** [comment]
It would be added only if the user provides the API key. So no privacy problems

**Comment by MartinKolarik** [comment]
@radulucut please add the user info into the location output like this:

![Image](https://github.com/user-attachments/assets/ad92ea6b-0805-42ad-9947-5dc3142e83f7)

To get the value, take all the probe tags, and:
 - filter for those that start with `u-` and do NOT contain a `:` character,
 - if more than 1 is left, find the largest common prefix of the values (e.g. `u-xx-1`, `u-xx-2` -> `u-xx`)

**Comment by MartinKolarik** [comment]
@Ernst2020 this feature will be included in the next release. It works as long as your probe has a custom user tag. We also added a setting to auto-tag all your probes, which you can enable at https://dash.globalping.io/settings ("Make my probes public").
